rgw/v2/tests/s3_swift/test_quota_management.py

```diff
@@ -0,0 +1,209 @@
+"""test_quota_management - Test quota management
+
+Usage: test_quota_management.py -c <input_yaml>
+
+<input_yaml>
+        Note: any one of these yamls can be used
+        test_quota_bucket_max_objects.yaml
+        test_quota_bucket_max_size.yaml
+        test_quota_user_max_objects.yaml
+        test_quota_user_max_size.yaml
+
+Operation:
+    Create non tenanted user
+    Create bucket
+    test bucket quota max objects
+    test bucket quota max size
+    test user quota max objects
+    test user quota max size
+"""
+import os
+import sys
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+import argparse
+import traceback
+
+import v2.lib.resource_op as s3lib
+import v2.utils.utils as utils
+from v2.lib.exceptions import RGWBaseException, TestExecError
+from v2.lib.resource_op import Config
+from v2.lib.s3.auth import Auth
+from v2.lib.s3.write_io_info import BasicIOInfoStructure, IOInfoInitialize
+from v2.tests.s3_swift import reusable
+from v2.tests.s3_swift.reusables import quota_management as quota_mgmt
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+
+TEST_DATA_PATH = None
+import logging
+
+log = logging.getLogger()
+
+
+def upload_object_initiate(
+    config, each_user, bucket, bucket_name, obj_name_suffix, obj_size
+):
+    config.obj_size = obj_size
+    s3_object_name = utils.gen_s3_object_name(bucket_name, obj_name_suffix)
+    log.info("s3 object name: %s" % s3_object_name)
+    s3_object_path = os.path.join(TEST_DATA_PATH, s3_object_name)
+    log.info("s3 object path: %s" % s3_object_path)
+    log.info("upload type: normal")
+    try:
+        reusable.upload_object(
+            s3_object_name,
+            bucket,
+            TEST_DATA_PATH,
+            config,
+            each_user,
+        )
+        return True
+    except TestExecError as e:
+        log.info(e)
+        return False
+
+
+def test_max_objects(quota_scope, config, each_user, bucket, max_objects):
```

Comment:
@hmaheswa , could we move this also to resuables/quota_management.py

Reply:
done madhavi